packages/emails/templates/team-invite-email.ts

```diff
@@ -23,10 +25,11 @@ export default class TeamInviteEmail extends BaseEmail {
   protected getNodeMailerPayload(): Record<string, unknown> {
     return {
       to: this.teamInviteEvent.to,
-      from: `Cal.com <${this.getMailerOptions().from}>`,
+      from: `${WEBSITE_DOMAIN} <${this.getMailerOptions().from}>`,
```

Comment:
I think this (and all emails), should use an App Name environment variable and/or previously suggested company name variable. 

For example I have it hosted at "booking.company.com" I wouldn't want that full domain in the subject of an email.

Reply:
I will change it to app name. Company name could be confusing if it is different from the application.